src/JJMasterData.Web/Areas/MasterData/Controllers/FileController.cs

```diff
@@ -0,0 +1,16 @@
+using JJMasterData.Commons.Util;
+using Microsoft.AspNetCore.Mvc;
+
+namespace JJMasterData.Web.Areas.MasterData.Controllers;
+
+[Area("MasterData")]
+public class FileController : MasterDataController
```

Comment:
Already exists, just moved out of FormController

Reply:
Great!